### PR TITLE
animals table pull request

### PR DESCRIPTION
**Pull request by theresetuyi**
## animal table
-Create tables in SQL.
-Insert and query data in SQL.
create Database
create table
Insert data
Select data

### CODE REVIEW

File: MIT.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (Qoosim) [inline MIT.md]: - [ ] This is the file that exists in your project and is the one you're supposed to reference in the MIT link not `./LICENSE` file whic does not exist. Kindly reference `./MIT` in the link. 

### 404 error
license

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id AUTO_INCREMENT,
```
Comment (Qoosim) [inline schema.sql]: - [ ] You did not specify the field type (i.e int) before you add the `AUTO_INCREMENT`. This is the error message I got while trying to create the animals table. You can use something similar to this :point_down: 

```
  id int not null AUTO_INCREMENT

  or 

  id serial
```

auto-increment

File: schema.sql [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+    name varchar(100),
+    date_of_bith DATA,
```
Comment (Qoosim) [inline schema.sql]: - [ ] There's a spelling error here. It is supposed to be `date_of_birth` NOT `date_of_bith`. Kindly look into it and fix.
- [ ] It also complains about the type `DATA`. It is supposed to the `date` not `data`. Kindly look into that and fix.


### Error
spelling-error

File: data.sql [hunk 4]
```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
```
Comment (Qoosim) [inline data.sql:3]: - [ ] There's a spelling error in the field `escap_attempts`. The animals table carries `escape_attemps` but you're referencing to `escape_attempts` while populating data into that field. Kindly check and fix it.

### Error
errro-2

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @theresetuyi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql [hunk 5]
```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escap_attempts, neutered, weight_kg)
```
Comment (gemmen29) [inline data.sql]: - [ ] Kindly fix this typo error in the insert statement for the `escap_attempts` as it's called `escape_attempt` in your schema.sql file.

File: queries.sql [hunk 6]
```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name  LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
```
Comment (gemmen29) [inline queries.sql:5]: - [ ] Kindly fix this typo error in the query `List the name of all animals that are neutered and have less than 3 escape attempts` for the `escap_attempts` as it's called `escape_attempt` in your schema.sql file.

File: queries.sql [hunk 7]
```diff
@@ -1,3 +1,11 @@
...
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
```
Comment (gemmen29) [inline queries.sql:7]: - [ ] Kindly fix this typo error in the query `List name and escape attempts of animals that weigh more than 10.5kg` for the `escap_attempts` as it's called `escape_attempt` in your schema.sql file.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @theresetuyi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Highlights💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @theresetuyi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**